lib/msal-node/README.md

```diff
@@ -29,32 +29,36 @@ MSAL Node enables applications to authenticate users using [Azure AD](https://do
 The current version supports the following ways of acquiring tokens:
 
 #### Public Client:
-- [Authorization Code Grant](https://oauth.net/2/grant-types/authorization-code/) with [PKCE](https://oauth.net/2/pkce/)
-- [Device Code Grant](https://oauth.net/2/grant-types/device-code/)
-- [Refresh Token Grant](https://oauth.net/2/grant-types/refresh-token/)
-- [Silent Flow](https://docs.microsoft.com/azure/active-directory/develop/msal-acquire-cache-tokens#acquiring-tokens-silently-from-the-cache)
-- [Username and Password flow](https://docs.microsoft.com/azure/active-directory/develop/msal-authentication-flows#usernamepassword)
+
+-   [Authorization Code Grant](https://oauth.net/2/grant-types/authorization-code/) with [PKCE](https://oauth.net/2/pkce/)
+-   [Device Code Grant](https://oauth.net/2/grant-types/device-code/)
+-   [Refresh Token Grant](https://oauth.net/2/grant-types/refresh-token/)
+-   [Silent Flow](https://docs.microsoft.com/azure/active-directory/develop/msal-acquire-cache-tokens#acquiring-tokens-silently-from-the-cache)
+-   [Username and Password flow](https://docs.microsoft.com/azure/active-directory/develop/msal-authentication-flows#usernamepassword)
 
 #### Confidential Client:
-- [Authorization Code Grant](https://oauth.net/2/grant-types/authorization-code/) with a client credential
-- [Refresh Token Grant](https://oauth.net/2/grant-types/refresh-token/)
-- [Silent Flow](https://docs.microsoft.com/azure/active-directory/develop/msal-acquire-cache-tokens#acquiring-tokens-silently-from-the-cache)
-- [Client Credential Grant](https://oauth.net/2/grant-types/client-credentials/)
-- [On-behalf-of flow](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-on-behalf-of-flow)
-- [Username and Password flow](https://docs.microsoft.com/azure/active-directory/develop/msal-authentication-flows#usernamepassword)
+
+-   [Authorization Code Grant](https://oauth.net/2/grant-types/authorization-code/) with a client credential
+-   [Refresh Token Grant](https://oauth.net/2/grant-types/refresh-token/)
+-   [Silent Flow](https://docs.microsoft.com/azure/active-directory/develop/msal-acquire-cache-tokens#acquiring-tokens-silently-from-the-cache)
+-   [Client Credential Grant](https://oauth.net/2/grant-types/client-credentials/)
+-   [On-behalf-of flow](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-on-behalf-of-flow)
+-   [Username and Password flow](https://docs.microsoft.com/azure/active-directory/develop/msal-authentication-flows#usernamepassword)
 
 **[Coming Soon]** In the future we plan to add support for:
-- [Integrated Windows Authentication flow](https://docs.microsoft.com/azure/active-directory/develop/msal-authentication-flows#integrated-windows-authentication)
+
+-   [Integrated Windows Authentication flow](https://docs.microsoft.com/azure/active-directory/develop/msal-authentication-flows#integrated-windows-authentication)
```

Comment:
These can probably be removed

Reply:
Just delete the whole thing or just the Coming soon?

Reply:
Lines 48-50